psutil/__init__.py

```diff
@@ -911,13 +911,15 @@ def num_threads(self):
         """Return the number of threads used by this process."""
         return self._proc.num_threads()
 
-    def threads(self):
-        """Return threads opened by process as a list of
-        (id, user_time, system_time) namedtuples representing
-        thread id and thread CPU times (user/system).
-        On OpenBSD this method requires root access.
-        """
-        return self._proc.threads()
+    if hasattr(_psplatform.Process, "threads"):
+
+        def threads(self):
+            """Return threads opened by process as a list of
+            (id, user_time, system_time) namedtuples representing
+            thread id and thread CPU times (user/system).
+            On OpenBSD this method requires root access.
+            """
+            return self._proc.threads()
```

Comment:
Same here. I would always provide this method and just have `_psaix.py` return an empty list.

Reply:
Actually forget about it. It is more correct to not provide the method. Let's just avoid mentioning this in the doc. The rest is fine and is good to go.